design-toolkit/components/src/components/clock/clock.stories.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { Clock } from '.';
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta = {
+  title: 'Components/Clock',
+  component: Clock,
+  args: {
+    className: undefined,
+    formatter: undefined,
+  },
+  argTypes: {
+    className: {
+      control: 'text',
+    },
+  },
+  parameters: {
+    controls: {
+      exclude: ['formatter'],
+    },
+  },
+} satisfies Meta<typeof Clock>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  render: Clock,
+};
```

Comment:
optional: it might be confusing for devs when first loading this story because it is black, unstyled text on a default dark background. you can fix this by overriding the theme decorator, something like this:

```suggestion
export const Default: Story = {
  render: Clock,
  decorators: [
    withThemeByClassName<ReactRenderer>({
      themes: {
        light: 'light !bg-surface-default',
        dark: 'dark !bg-surface-default',
      },
      defaultTheme: 'light',
    }),
  ],
};
```

then again, we specifically say it is *unstyled* text so I can be convinced it is unnecessary.

Reply:
this change caused some weird behaviors with light/dark mode behavior, so we're skipping it.